subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -81,6 +81,20 @@ def init_test_specific_variables(self):
         raise NotImplementedError("Test specific variables has to be "
                                   "overridden by child.")
 
+    def _wait_for_ready(self):
```

Comment:
Could this method be replaced by a call to ``output.wait_for_output()``?

(I know sometimes it doesn't always fit well)